app/assets/src/components/common/Header.jsx

```diff
@@ -4,7 +4,7 @@ import { forbidExtraProps } from "airbnb-prop-types";
 import cx from "classnames";
 
 import { RequestContext } from "~/components/common/RequestContext";
-import ToastContainer from "~ui/containers/ToastContainer";
+import ToastContainer from "~/components/common/ToastContainer";
```

Comment:
Why the move? It is a UI element...

Reply:
`<ToastContainer>` can only be rendered once in the application. I felt like it belonged with `<Header>`, which is also in `~/components/common`. It felt confusing to put it in `/ui`, because it's not re-usable.

Reply:
I think of it more like what would happen if we wanted to package the IDseq UI. In that case, `ToastContainer` would need to be included... that it is only instantiated once, is just a consequence of the type of element.